pkg/registry/core/service/storage/storage_test.go

```diff
@@ -157,7 +158,21 @@ func newStorageWithPods(t *testing.T, ipFamilies []api.IPFamily, pods []api.Pod,
 	if err != nil {
 		t.Fatalf("unexpected error from REST storage: %v", err)
 	}
-	return serviceStorage, statusStorage, server
+	return &TestREST{serviceStorage}, statusStorage, server
```

Comment:
do we need `TestREST` to be exportable?

Reply:
Good point.  Will rename and make it more obvious.